### ISSUE TITLE
Elements lacking adequate names / alt text.

### ISSUE THREAD

**Issue by ConnorTurnbull** [ConnorTurnbull/Project-4#6]
- Elements lacking adequate names or ALT text can cause issues for users that require the use of screen readers.

**Comment by ConnorTurnbull** [comment]
- Social media icons/links on home page do not have naming text.

![image](https://user-images.githubusercontent.com/110614970/209553782-e468efea-c2d1-4c83-bae8-e533e213696b.png)
 
- Added "aria-label" attributes to all social media icons, as found here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label
 -Note that the social media icons are included in the HTML for page2 but not visible on page itself.

Home page before:

![image](https://user-images.githubusercontent.com/110614970/209558900-15817813-1715-4dd5-9dc5-5363d8084df8.png)

![image](https://user-images.githubusercontent.com/110614970/209558915-2ee28c89-a3fe-49a1-a50b-d1cf05e9a0b5.png)

Home page after:

![image](https://user-images.githubusercontent.com/110614970/209558937-d49a2203-8ea8-44fe-9a07-47f8b11ecc39.png)

![image](https://user-images.githubusercontent.com/110614970/209558948-cd183df1-3e95-499d-a89d-56f617537c03.png)

Page2 before:

![image](https://user-images.githubusercontent.com/110614970/209558969-e2053de2-518b-4cf9-8063-57d7fd8a0b76.png)

![image](https://user-images.githubusercontent.com/110614970/209558978-586cce09-5961-4cd6-97fa-cf688134fa99.png)

**Comment by ConnorTurnbull** [comment]
- Added "aria-label" attributes to all social media icons, as found here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label

Before:

![image](https://user-images.githubusercontent.com/110614970/209554514-94b684e9-0a3f-44b2-bb1a-f0cf83336432.png)

After:

![image](https://user-images.githubusercontent.com/110614970/209554538-f7b129c5-c261-4b90-8a2b-0aff51c69c4e.png)

**Comment by ConnorTurnbull** [comment]
- Header logo on page2 does not have name or alt text.

![image](https://user-images.githubusercontent.com/110614970/209557826-6af49349-f1d4-402c-b5fa-00a22005c968.png)

- Copied alt text attribute from home page HTML to page2.

Home page:

![image](https://user-images.githubusercontent.com/110614970/209559048-4b091985-a620-482f-b351-55971578f9f9.png)

Page2:

![image](https://user-images.githubusercontent.com/110614970/209559062-46580236-d18f-46a9-a542-605bef742d55.png)

![image](https://user-images.githubusercontent.com/110614970/209559072-89ca4c33-5a5d-44e4-bae0-7a725d0e5e00.png)


Page2 before:

![image](https://user-images.githubusercontent.com/110614970/209559114-c6cb132c-ba14-4eb5-b701-cf5b863e0f4a.png)

Page2 after:

![image](https://user-images.githubusercontent.com/110614970/209559168-72cbae0e-b7ae-4140-b2f3-1bb55e1f0898.png)

**Comment by ConnorTurnbull** [comment]
- Copied alt text attribute from home page HTML to page2.

Home page:

![image](https://user-images.githubusercontent.com/110614970/209557906-7be3d2fd-dd8d-4ec9-b219-d849f21a3cb0.png)

Page2:

![image](https://user-images.githubusercontent.com/110614970/209557956-c8a2b3fe-9234-4f7c-bfc1-8494889b84a3.png)

**Comment by ConnorTurnbull** [comment]
- Page2 form elements do not have labels.

![image](https://user-images.githubusercontent.com/110614970/209560972-ee4f3d94-46a7-4f04-924b-27994aa944b3.png)


- Added aria-label attributes to form elements as per previous fix of similar issue.

Before:

![image](https://user-images.githubusercontent.com/110614970/209561144-ded1547a-2fb8-44b9-8cc4-97413ff6ba97.png)

After:

![image](https://user-images.githubusercontent.com/110614970/209561199-23f511d9-8b2e-4398-ab82-8551fb2f7b60.png)


Page2 before:

![image](https://user-images.githubusercontent.com/110614970/209561258-e4ca812e-7063-4923-9ef1-d74157a65e94.png)

Page2 after:

![image](https://user-images.githubusercontent.com/110614970/209561328-e92beb64-8bd0-4d45-891a-e57444a4216d.png)

**Comment by ConnorTurnbull** [comment]
- Lighthouse reports

[#6 fix - home.pdf](https://github.com/ConnorTurnbull/Project-4/files/10303928/6.fix.-.home.pdf)
[#6 fix - page2.pdf](https://github.com/ConnorTurnbull/Project-4/files/10303927/6.fix.-.page2.pdf)
